### PR TITLE
OCM-11000 | ci: Automate ids:67149,76200

### PR DESCRIPTION
**Pull request by radtriste**
https://issues.redhat.com/browse/OCM-11000

### CODE REVIEW

**Comment by radtriste** [comment]
@OAharoni-RedHat @jfrazierRH could you check that one ?

File: tests/e2e/hcp_machine_pool_test.go [hunk 1]
```diff
@@ -575,5 +577,170 @@ var _ = Describe("HCP Machine Pool", labels.Feature.Machinepool, func() {
...
+					clusterConfig.Version.RawID,
+					"",
+					"")
+				Expect(err).To(HaveOccurred())
+				Expect(err.Error()).To(ContainSubstring(fmt.Sprintf("Cluster '%s' is not yet ready", notReadyClusterID)))
+			} else {
+				Logger.Info("No cluster found in state not-ready. Skipping this step")
+			}
+
+			By("Find a version has available upgrade versions")
```
Comment (jfrazierRH) [inline tests/e2e/hcp_machine_pool_test.go:638]: For each of these blocks, would it make sense to have a single execution statement and that loops through passed in flags? 

Something similar to this: https://github.com/openshift/rosa/blob/3a9ac1fa489b6d6595f43338c2674af2d463a5ea/tests/e2e/test_rosacli_node_pool.go#L1150

Comment (radtriste) [reply tests/e2e/hcp_machine_pool_test.go:638]: We are in this case testing different cli options and we use either `CreateAutomaticUpgrade` or `CreateManualUpgrade` method.
I would see it as not readable to have a huge map containing all cases.

File: tests/e2e/test_rosacli_machine_pool.go [hunk 2]
```diff
@@ -1426,3 +1426,74 @@ var _ = Describe("Edit machinepool",
...
+			}
+
+		})
+
+		AfterEach(func() {
+			By("Clean remaining resources")
+			rosaClient.CleanResources(clusterID)
+		})
+
+		It("will fail - [id:76200]",
```
[the remark this answers is not in the document]
Comment (radtriste) [reply tests/e2e/test_rosacli_machine_pool.go]: updated

File: tests/e2e/hcp_machine_pool_test.go [hunk 3]
```diff
@@ -575,5 +577,170 @@ var _ = Describe("HCP Machine Pool", labels.Feature.Machinepool, func() {
...
+				_, err = machinePoolUpgradeService.CreateManualUpgrade(
+					notReadyClusterID,
+					"anything",
+					clusterConfig.Version.RawID,
+					"",
+					"")
+				Expect(err).To(HaveOccurred())
+				Expect(err.Error()).To(ContainSubstring(fmt.Sprintf("Cluster '%s' is not yet ready", notReadyClusterID)))
+			} else {
+				Logger.Info("No cluster found in state not-ready. Skipping this step")
```
Comment (OAharoni-RedHat) [inline tests/e2e/hcp_machine_pool_test.go:635]: How often do we expect a cluster to be available in a non-ready step?

Comment (radtriste) [reply tests/e2e/hcp_machine_pool_test.go:635]: non ready can be error/installing/uninstalling... so there should be quite many

**Comment by radtriste** [comment]
test failures are not related to current change

**Pull request merged**